frontend/src/my-knolist/NewProjectModal.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Modal, Button, Form, FormGroup, ControlLabel, HelpBlock, FormControl } from "rsuite";
+
+function NewProjectModal(props) {
+  const show = props.show;
+  const setShow = props.setShow;
+
+  return (
+    <Modal show={show} style={{ overflow: "hidden" }} onHide={() => setShow(false)}>
+      <Modal.Header><Modal.Title>New Project</Modal.Title></Modal.Header>
+      <Modal.Body>
+        <Form fluid>
+          <FormGroup>
+            <ControlLabel>Title</ControlLabel>
+            <FormControl name="title" />
+            <HelpBlock>Required</HelpBlock>
+          </FormGroup>
+          <FormGroup>
+            <ControlLabel>Description</ControlLabel>
+            <FormControl
+              rows={5}
+              name="description"
+              componentClass="textarea" />
+          </FormGroup>
+        </Form>
+      </Modal.Body>
+      <Modal.Footer>
+        <Button onClick={() => setShow(false)} appearance="primary">
+          Ok
```

Comment:
Could we use `Create` instead of `Ok`? I think it would be a bit better in terms of UX
```suggestion
          Create
```